README.md

```diff
@@ -13,108 +13,27 @@ Storybook documentation
 
 | Table of Contents |                                                                                     |
 | ----------------- | ----------------------------------------------------------------------------------- |
-| 1.                | [Contributing Quickstart](#contributing-quickstart)                                 |
-| 2.                | [Developing with NPM Link](#developing-with-npm-link)                               |
-| 3.                | [Using the Design System in Your Product](#using-the-design-system-in-your-product) |
-| 4.                | [Using Chakra UI Components](#using-chakra-ui-components)                           |
-| 5.                | [Storybook](#storybook)                                                             |
-| 6.                | [Typescript Usage](#typescript-usage)                                               |
-| 7.                | [Unit Testing](#unit-testing)                                                       |
-| 8.                | [Accessibility](#accessibility)                                                     |
+| 1.                | [Using the Design System in Your Product](#using-the-design-system-in-your-product) |
+| 2.                | [Using Chakra UI Components](#using-chakra-ui-components)                           |
+| 3.                | [Storybook](#storybook)                                                             |
+| 4.                | [Accessibility](#accessibility)                                                     |
+| 5.                | [Contributing Quickstart](#contributing-quickstart)                                 |
```

Comment:
Moved the contributing info below all the "how to use" sections. Most devs reading the README will be interested in using it first before contributing back.